test/uk/gov/hmrc/merchandiseinbaggage/pagespecs/GoodsInVehiclePageSpec.scala

```diff
@@ -25,7 +25,7 @@ import uk.gov.hmrc.merchandiseinbaggage.pagespecs.pages.{CheckYourAnswersPage, J
 class GoodsInVehiclePageSpec extends DeclarationDataCapturePageSpec[YesNo, RadioButtonPage[YesNo]] {
   override lazy val page: RadioButtonPage[YesNo] = wire[RadioButtonPage[YesNo]]
 
-  private val requiredAnswerValidationMessage = "Select yes if you are travelling by vehicle"
+  private val requiredAnswerValidationMessage = "Select yes if the goods are arriving by vehicle"
```

Comment:
can we start using messages `val messageApi: Map[String, String] = messagesApi.messages("default")` instead of hard coded string - also we will make changes only in messages file